qtaf-xray-plugin/src/main/java/de/qytera/qtaf/xray/repository/xray/XrayTestRepository.java

```diff
@@ -0,0 +1,87 @@
+package de.qytera.qtaf.xray.repository.xray;
+
+import de.qytera.qtaf.core.QtafFactory;
+import de.qytera.qtaf.core.config.exception.MissingConfigurationValueException;
+import de.qytera.qtaf.core.gson.GsonFactory;
+import de.qytera.qtaf.http.RequestBuilder;
+import de.qytera.qtaf.http.WebService;
+import de.qytera.qtaf.xray.config.XrayConfigHelper;
+import de.qytera.qtaf.xray.dto.request.xray.ImportExecutionResultsRequestDto;
+import de.qytera.qtaf.xray.dto.response.steps.XrayTestStepResponseDto;
+import de.qytera.qtaf.xray.dto.response.xray.ImportExecutionResultsResponseCloudDto;
+import de.qytera.qtaf.xray.dto.response.xray.ImportExecutionResultsResponseDto;
+import de.qytera.qtaf.xray.dto.response.xray.ImportExecutionResultsResponseServerDto;
+import de.qytera.qtaf.xray.events.XrayEvents;
+import jakarta.ws.rs.core.HttpHeaders;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Collection;
+import java.util.Map;
+
+public interface XrayTestRepository extends XrayEndpoint {
+
```

Comment:
Please add DocString

Reply:
Fixed in [7782f10](https://github.com/Qytera-Gmbh/QTAF/pull/89/commits/7782f10a956c9199bb863187a388462b6c0a5224).